docs/source/technical_tutorials/visualization/visualize_mosaic_stac_ipyleaflet.ipynb

```diff
@@ -5,40 +5,95 @@
    "id": "a23add21-c317-473e-b8c5-bc438ad60d3c",
```

Comment:
<p>Maybe we can use the capitalized version for the title, like MosaicJSON?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>